bigframes/core/blocks.py

```diff
@@ -679,6 +680,7 @@ def to_pandas_batches(
         page_size: Optional[int] = None,
         max_results: Optional[int] = None,
         allow_large_results: Optional[bool] = None,
+        callback: Callable = lambda _: None,
```

Comment:
Per discussion offline, we may want to provide a global progress publisher object rather than add callbacks to each method that needs them.